config/environments/test.rb

```diff
@@ -38,11 +38,11 @@
   # config.action_view.raise_on_missing_translations = true
 
   config.paperclip_defaults = {
-    :storage => :s3,
-    :s3_credentials => {
-      :bucket => ENV['S3_BUCKET_NAME'],
-      :access_key_id => ENV['AWS_ACCESS_KEY_ID'],
-      :secret_access_key => ENV['AWS_SECRET_ACCESS_KEY']
+    storage: :s3,
+    s3_credentials: {
+      bucket: ENV['S3_TEST_BUCKET_NAME'],
+      access_key_id: ENV['AWS_TEST_ACCESS_KEY_ID'],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.